spec/lib/salus/scanners/report_go_dep_spec.rb

```diff
@@ -48,6 +48,43 @@
     end
   end
 
+  describe '#record_dep_from_go_mod' do
```

Comment:
💯 I think it would be a great opportunity to add some of these test to confirm if `go list` can handle
- internal modules
- both go.mod and go.sum
- if it accounts for transitive dependencies

Reply:
Good points! 

I think after testing extensively it seems that internal modules were a lot of trouble to deal with when using `go list` so instead I just parsed go.sum / go.mod manually in addition to the existing Gopkg.lock as were thinking about doing before.